core-war/src/main/java/org/silverpeas/web/importexport/servlets/ImportExportRequestRouter.java

```diff
@@ -107,15 +108,15 @@ public String getDestination(String function, ImportExportSessionController impo
         destination = "/importExportPeas/jsp/selectExportMode.jsp";
       } else if ("ExportSavedItems".equals(function)) {
         String mode = request.getParameter("ExportMode");
-        importExportSC.processExportOfSavedItems(mode);
+        importExportSC.processExportOfSavedItems(mode, isNameUsedForFolders(request));
         destination = "/importExportPeas/jsp/pingExport.jsp";
       } else if ("ExportItems".equals(function)) {
         @SuppressWarnings("unchecked")
         List<WAAttributeValuePair> itemPKs = (List<WAAttributeValuePair>) request.getAttribute(
             "selectedResultsWa");
         NodePK rootPK = (NodePK) request.getAttribute("RootPK");
         if (itemPKs != null && !itemPKs.isEmpty()) {
-          importExportSC.processExport(itemPKs, rootPK);
+          importExportSC.processExport(itemPKs, rootPK, true);
```

Comment:
Why here the parameter is set explicitly to `true` instead of using the `isNameUsedForFolders(request) ` method?

Reply:
I have to check this stuff...

Reply:
Here, this action is called from the search engine results. There is no popup to choose the export mode, it's FULL export by default. So we have the choice between true or false...

Reply:
ok